app/include/dt-bindings/zmk/macros.h

```diff
@@ -0,0 +1,12 @@
+
+#define MAC_TAP_BIT (1 << 31)
+#define MAC_PRESS_BIT (1 << 30)
+#define MAC_REL_BIT (1 << 29)
+
+#define MAC_TAP_WAIT(ms) (MAC_TAP_BIT + ms)
+#define MAC_PRESS_WAIT(ms) (MAC_PRESS_BIT + ms)
+#define MAC_REL_WAIT(ms) (MAC_REL_BIT + ms)
+
+#define MAC_TAP MAC_TAP_WAIT(100)
```

Comment:
If you want to be able to override the default delay, you could maybe use `MAC_REL_BIT - 1` as a special "default" value that gets swapped out for a DT property value when the macros code finds it.